parse.py

```diff
@@ -138,6 +138,17 @@ def process_enc_line(line, ext):
 
     return (name, single_dict)
 
+def same_ext (ext_name, ext_name_list):
+  type1 = ext_name.split("_")[0]
+  for ext_name1 in ext_name_list:
+    type2 = ext_name1.split("_")[0]
+    # "rv" mean insn for rv32 and rv64
```

Comment:
How about rv128
https://github.com/riscv/riscv-opcodes/blob/master/unratified/rv128_c

Reply:
From the readme's [File Naming Policy](https://github.com/riscv/riscv-opcodes#file-naming-policy), instructions in the file name with the `rv_` prefix support only rv32 and rv64.

Reply:
Oh, I misunderstood the logic here. Thanks for pointing out!